proto/storage/compliance_integration.proto

```diff
@@ -7,13 +7,12 @@ import "gogoproto/gogo.proto";
 
 package storage;
 
-// Next Tag: 7
+// Next Tag: 6
 message ComplianceIntegration {
     string id                     = 1 [(gogoproto.moretags) = 'sql:"pk,type(uuid)"'];
     string version                = 2 [(gogoproto.moretags) = 'search:"Compliance Operator Version,hidden,store"'];
     string cluster_id             = 3 [(gogoproto.moretags) = 'search:"Cluster ID,hidden,store" sql:"fk(Cluster:id),no-fk-constraint,type(uuid),index=category:unique;name:compliance_unique_indicator"'];
-    string namespace              = 4 [(gogoproto.moretags) = 'search:"Namespace,store"'];
-    string namespace_id           = 6 [(gogoproto.moretags) = 'search:"Namespace ID" sql:"fk(NamespaceMetadata:id),no-fk-constraint,type(uuid)"'];
+    string compliance_namespace   = 4;
```

Comment:
this is just tracking the namespace compliance operator is installed in.  Leaving as namespace brought it into SAC which was not the intent of this field.